addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1905,6 +1904,37 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.Accordion = options.Class.extend({
+    /**
+     * Toggle the position of the cards' collapse state icons
+     * (either floats left (default) or right)
+     *
+     * @see this.selectClass for parameters
+     */
+    iconPos: function (previewMode, widgetValue, params) {
+        const $cards = this.$('.accordion .card');
+        if (widgetValue == "right") {
```

Comment:
Add a linter to your editor so you can respect the guidelines and avoid making syntax/whatever mistakes. There is a sample config file for ESLint here: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines but you can install whatever linter you want.

On this line for example, you are supposed to use `===` instead of `==` and your editor should have warned you about it.